tidb-cloud/tidb-cloud-org-sso-authentication.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Organization SSO Authentication
+summary: Learn how to log in to the TiDB Cloud console via your customized organization authentication.
+---
+
+# Organization SSO Authentication
+
+Single Sign-On (SSO) is an authentication scheme that allows members in your TiDB Cloud [organization](/tidb-cloud/tidb-cloud-glossary.md#organization) to log into TiDB Cloud with identities from an identity providers (IdP) instead of emails and passwords.
+
+TiDB Cloud supports the following two types of SSO authentication:
+
+- [Basic SSO](/tidb-cloud/tidb-cloud-org-sso-authentication.md): members can log in to the [TiDB Cloud console](https://tidbcloud.com/) using their GitHub, Google, or Microsoft authentication methods. The basic SSO is enabled by default for all organizations in TiDB Cloud.
+
+- Cloud Organization SSO: members can log in to a custom login page of TiDB Cloud using the authentication methods specified by your organization. The Cloud Organization SSO is disabled by default.
+
+Compared with basic SSO, Cloud Organization SSO provides more flexibility and customization so you can better meet your organization's security and compliance requirements. For example, you can specify which authentication methods are displayed on the login page, limit which email address domains are allowed for login, and let your members to log into TiDB Cloud with your identity provider (IdP) that uses OpenID Connect (OIDC) identity protocols.
+
+In this document, you will learn how to migrate the authentication scheme of your organization from basic SSO to Cloud Organization SSO.
+
+> **Note:**
+>
+> - The Cloud Organization SSO feature is in beta and is only available upon request. To request this feature, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com) and click **Chat with Us**. Then, fill in "Apply for Cloud Organization SSO" in the **Description** field and click **Send**.
+> - If your current TiDB login URL starts with `https://tidbcloud.com/enterprise/`, it means that Cloud Organization SSO is already enabled for your organization.
+
+## Before you begin
+
+Before migrating to Cloud Organization SSO, check and confirm the items in this section for your organization.
+
+> **Note:**
+>
+> - Once Cloud Organization SSO is enabled, it cannot be disabled.
+> - To enable Cloud Organization SSO, you need to have the owner role in your TiDB Cloud organization. For more information about roles, see [Manage role access](/tidb-cloud/manage-user-access.md#manage-role-access).
+
+### Decide a custom URL for the TiDB Cloud login page of your organization
+
+When Cloud Organization SSO is enabled, your members must use your custom URL instead of the public login URL at `https://tidbcloud.com` to log into TiDB Cloud.
```

Comment:
```suggestion
When Cloud Organization SSO is enabled, your members must use your custom URL instead of the public login URL (`https://tidbcloud.com`) to log into TiDB Cloud.
```